orchestrator/src/main/java/com/yahoo/vespa/orchestrator/OrchestratorImpl.java

```diff
@@ -427,10 +425,7 @@ public boolean isQuiescent(ApplicationId id) {
                 ClusterControllerClient client = clusterControllerClientFactory.createClient(clusterControllers, cluster.clusterId().s());
                 for (ServiceInstance service : cluster.serviceInstances()) {
                     try {
-                        ClusterControllerStateResponse response = client.setNodeState(context,
-                                                                                      VespaModelUtil.getStorageNodeIndex(service.configId()),
-                                                                                      MAINTENANCE);
-                        if ( ! response.wasModified)
+                        if ( ! client.setNodeState(context, service.hostName(), VespaModelUtil.getStorageNodeIndex(service.configId()), MAINTENANCE))
```

Comment:
isQuiescent() is not used except by tests.  Was this something we used for CC migration that we no longer need?  It is a nice method though.

Reply:
Hmm ... true, it was.